exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -0,0 +1,473 @@
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.function.BinaryOperator;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertNotNull;
+import static junit.framework.TestCase.assertTrue;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class ListOpsTest {
+
+    private static final List<Integer> EMPTY_LIST
+            = Collections.emptyList();
+
+    @Test
+    public void lengthOfAnEmptyListShouldBeZero() {
+        final int expected = 0;
+        final int actual = ListOps.length(EMPTY_LIST);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldReturnCorrectLengthOfAnNonEmptyList() {
+        final int expected = 4;
+        final List<Integer> list = Collections.unmodifiableList(
+                IntStream.range(0, expected)
+                        .boxed()
+                        .collect(Collectors.toList())
+        );
+        final int actual = ListOps.length(list);
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    @Ignore
+    public void shouldReverseAnEmptyList() {
+        final List<Integer> actual = ListOps.reverse(EMPTY_LIST);
+
+        assertNotNull(actual);
+        assertTrue(actual.isEmpty());
+    }
+
+    @Test
+    @Ignore
+    public void shouldReverseNonEmptyList() {
+        final int length = 100;
+        final int startValue = 0;
+        final List<Integer> reversedList
+                = IntStream.range(startValue, length)
+                        .boxed()
+                        .collect(Collectors.toList());
+        Collections.reverse(reversedList);
+        final List<Integer> expected
+                = Collections.unmodifiableList(reversedList);
```

Comment:
Seems that it's unnecessary as we don't pass it anywhere. That's an example where code reviews really shine. 👍